contracts/crowdsale/distribution/PostDeliveryCrowdsale.sol

```diff
@@ -12,19 +12,26 @@ import "../../math/SafeMath.sol";
 contract PostDeliveryCrowdsale is TimedCrowdsale {
   using SafeMath for uint256;
 
-  mapping(address => uint256) public balances;
+  mapping(address => uint256) private balances_;
 
   /**
    * @dev Withdraw tokens only after crowdsale ends.
    */
   function withdrawTokens() public {
     require(hasClosed());
-    uint256 amount = balances[msg.sender];
+    uint256 amount = balances_[msg.sender];
     require(amount > 0);
-    balances[msg.sender] = 0;
+    balances_[msg.sender] = 0;
     _deliverTokens(msg.sender, amount);
   }
 
+  /**
+   * @return the balance of an account.
+   */
+  function balanceOf(address _account) public view returns(uint256) {
```

Comment:
You see what I did here? :o uncle bob martin is crying.

Reply:
:joy: